pyoverkiz/clients/somfy_local.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+from typing import Any
+
+from pyoverkiz.clients.overkiz import OverkizClient
+from pyoverkiz.types import JSON
+
+
+class SomfyLocalClient(OverkizClient):
+    async def _login(self, username: str, password: str) -> bool:
+        """There is no login for Somfy Local"""
+        return True
```

Comment:
Should we not make an alternative call here? Can we just return True while we aren't sure if the token is correct?

Reply:
Well, this login method should not be called when you used the Local API. I can instead raise an exception too make this more clear.